common/lib/xmodule/xmodule/graders.py

```diff
@@ -395,11 +395,11 @@ def grade(self, grade_sheet, generate_random_scores=False):
                 )
             else:
                 percentage = 0.0
-                summary = u"{section_type} {index} Unreleased - 0% (?/?)".format(
+                # Translator's example: Homework 1 - Unreleased - 0% (?/?)
```

Comment:
It might be good to explain what unreleased means in this comment.

Reply:
Also what `(?/?)` is.

Reply:
`(?/?)` is a literal string. If the information was known it would like something like `(99/100)`

Reply:
Also, I think you need to prefix these comments with `Translators:` (not `Translator's example`) in order for gettext to pick it up and add it to the .po file automatically.

Reply:
Updated.